eq-author/src/App/page/Design/Validation/AnswerValidation.js

```diff
@@ -228,49 +225,4 @@ UnwrappedAnswerValidation.propTypes = {
   answer: CustomPropTypes.answer,
 };
 
-export const VALIDATION_QUERY = gql`
-  subscription Validation($id: ID!) {
-    validationUpdated(id: $id) {
-      id
-      totalErrorCount
-      sections {
-        pages {
-          ... on QuestionPage {
-            id
-            validationErrorInfo {
-              id
-              totalCount
-            }
-            answers {
-              ... on BasicAnswer {
-                id
-                validation {
-                  ... on NumberValidation {
-                    minValue {
-                      id
-                      validationErrorInfo {
-                        ...ValidationErrorInfo
-                      }
-                    }
-                    maxValue {
-                      id
-                      validationErrorInfo {
-                        ...ValidationErrorInfo
-                      }
-                    }
-                  }
-                }
-              }
-              ... on MultipleChoiceAnswer {
-                id
-              }
-            }
-          }
-        }
-      }
-    }
-  }
-  ${ValidationErrorInfo}
-`;
-
-export default withValidations(UnwrappedAnswerValidation, VALIDATION_QUERY);
+export default UnwrappedAnswerValidation;
```

Comment:
Should this be called `AnswerValidation` instead? As it is no longer being wrapped.